packages/react/src/ActionList/Item.tsx

```diff
@@ -272,20 +292,31 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
       id: itemId,
     }
 
-    const containerProps = _PrivateItemWrapper ? {role: itemRole ? 'none' : undefined} : menuItemProps
+    const containerProps = _PrivateItemWrapper
+      ? {role: itemRole ? 'none' : undefined, ...props}
+      : // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+        (validRole && {...menuItemProps, ...props, ref: forwardedRef}) || {}
 
-    const wrapperProps = _PrivateItemWrapper ? menuItemProps : {}
+    const wrapperProps = _PrivateItemWrapper
+      ? menuItemProps
+      : !validRole && {
+          ...menuItemProps,
+          ...props,
+          styles: merge<BetterSystemStyleObject>(styles, sxProp),
+          ref: forwardedRef,
+        }
 
     return (
       <ItemContext.Provider
         value={{variant, disabled, inactive: Boolean(inactiveText), inlineDescriptionId, blockDescriptionId}}
       >
         <LiBox
-          ref={forwardedRef}
-          sx={merge<BetterSystemStyleObject>(styles, sxProp)}
+          sx={merge<BetterSystemStyleObject>(
+            validRole || _PrivateItemWrapper ? styles : listItemStyles,
+            validRole || _PrivateItemWrapper ? sxProp : {},
+          )}
```

Comment:
I was having issues with the styles in other components that use `ActionList` (e.g. `ActionMenu`, `NavList`), which is why we have this conditional. Ensures that styles are properly passed to `<LiBox>` depending on if it's a `validRole`, or `_PrivateItemWrapper`.